src/uct/ib/rc/accel/rc_mlx5_iface.c

```diff
@@ -455,10 +481,12 @@ UCS_CLASS_INIT_FUNC(uct_rc_mlx5_iface_common_t,
         goto cleanup_stats;
     }
 
-    status = uct_ib_mlx5_srq_init(&self->rx.srq, self->super.rx.srq.srq,
-                                  self->super.super.config.seg_size);
-    if (status != UCS_OK) {
-        goto cleanup_tm;
+    if (self->rx.srq.type == UCT_IB_MLX5_OBJ_TYPE_VERBS) {
+        status = uct_ib_mlx5_srq_init(&self->rx.srq, self->rx.srq.verbs.srq,
```

Comment:
why only for verbs? can't we do same for devx?

Reply:
Its uct_ib_mlx5dv_init_obj() wrapper, N/A for devx

Reply:
so maybe move it to uct_rc_mlx5_init_rx()?